CONTRIBUTING.md

```diff
@@ -1,45 +1,53 @@
 # How to contribute to Fury
```

Comment:
We can polish more espcially for those handy scripts. But let's first formatting and tidy.